src/assets/fonts/text-security/package.json

```diff
@@ -0,0 +1,46 @@
+{
```

Comment:
Better to find a way install it as a normal package.
If you are worried about 
```
<link href="node_modules/text-security/text-security.css" rel="stylesheet" type="text/css">
```
in index.html, then I am 65% sure webpack will properly process such urls. You can test it by doing `yarn build` and checking `index.html` in `dist` folder.

Alternatively, you could try importing css file via `@import` in `index.html`.